layouts/partials/header.html

```diff
@@ -27,32 +27,32 @@ <h4>{{ .Site.Params.description }}</h4>
 			{{ $icon_size := "64px" }}
 			{{ if .Site.Params.email }}
 			<a href="mailto:{{ .Site.Params.email }}" title="Email">
-				<img alt="Email" height="{{ $icon_size }}" width="{{ $icon_size }}" src="{{ "/images/email.svg" | relURL}}" />
+				<img alt="Email" height="{{ $icon_size }}" width="{{ $icon_size }}" src="{{ "/images/icons/email.svg" | relURL}}" />
```

Comment:
I like the idea of being more organized with the static assets. If you move the corresponding files from ./static/images in this repo

Reply:
Yep. Actually I did that at later stage because as you mentioned being organized is a good. Thanks.